client/my-sites/themes/theme-errors.jsx

```diff
@@ -8,10 +8,14 @@ import wpcom from 'calypso/lib/wp';
  * avoid querying the WP.com cache of a site, which returns theme
  * errors for the multisite (caused by its 'force=wpcom' param).
  */
-async function querySiteDataWithoutForceWpcom( siteId ) {
-	return await wpcom.req.get( {
+function querySiteDataWithoutForceWpcom( siteId ) {
+	return wpcom.req.get( {
 		path: '/sites/' + encodeURIComponent( siteId ),
 		apiVersion: '1.1',
+		query: {
+			fields: 'ID,options',
+			options: 'theme_errors',
```

Comment:
Completely forgot that the endpoint accepted these queries - thanks for tidying this up!